Multisig/Assets.xcassets/ico-info.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
You can use `info.circle` system icon

Reply:
it's a different icon with a different look. I would not use it. Instead, I have created dark mode version for icon from our design and added it to the project.

Reply:
![image](https://user-images.githubusercontent.com/17750984/147557061-1cab0561-d726-4621-b133-c83ee66a3b49.png)